Cargo.toml

```diff
@@ -100,6 +100,7 @@ displaydoc = "0.2.5"
 edgehog-containers = { package = "edgehog-device-runtime-containers", path = "./edgehog-device-runtime-containers", version = "=0.1.0" }
 edgehog-device-forwarder-proto = "0.1.0"
 edgehog-forwarder = { package = "edgehog-device-runtime-forwarder", path = "./edgehog-device-runtime-forwarder", version = "=0.8.1" }
+eyre = "0.6.12"
```

Comment:
Don't we already have `stable_eyre` & `color_eyre`?

Reply:
Yes, but those are used in the binary and in the e2e-test. While eyre can be used in the library, since it doesn't provide a panic hook. It provides the report type to print the full error report.